src/profile-logic/call-tree.js

```diff
@@ -100,8 +100,7 @@ export class CallTree {
    */
   preloadChildrenCache() {
     if (!this._isChildrenCachePreloaded) {
-      this._children.clear();
-      this._children.set(-1, []); // -1 is the parent of the roots
+      this._children[-1] = []; // -1 is the parent of the roots
```

Comment:
Note that "-1" is converted to a string here and is added as an extra property. This works because this is JavaScript... but please add a comment explaining it :)